### DISCUSSION TITLE
Is there a way to watch lineage event for Hive execution?

### DISCUSSION THREAD

**Discussion by joint-song** [apache/kyuubi#6705] [Q&A]
There's an `OperationLineageKyuubiEvent` event which contains inputs and outputs, but it's located under *kyuubi-spark-lineage* package. Is this only available for Spark sql? How can I watch lineage event for Hive sql? Or can I use customized hive hooks in kyuubi server?

**Comment by wForget** [comment]
> Or can I use customized hive hooks in kyuubi server?

Yes, you can do that, you just need to configure hive hook in hive-site.xml and place the hook jar path in hive engine classpath.
